front-end/src/screens/MenuBar/index.tsx

```diff
@@ -127,12 +127,13 @@ const MenuBar = ({ className } : Props): JSX.Element => {
 export default styled(MenuBar)`
 	&.ui.menu, .ui.inverted.menu {
 		font-family: 'Roboto Mono';
+		background: #000;
```

Comment:
This should be a variable. For another PR, the font-family should also not be hardcoded IMO